internal/utils/test_case.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Better to put the file under testdata/. See https://dave.cheney.net/2016/05/10/test-fixtures-in-go

Reply:
I have reorganized the additional files needed for the test.